relay-server/src/envelope.rs

```diff
@@ -509,6 +513,11 @@ pub struct ItemHeaders {
     #[serde(default, skip)]
     replay_combined_payload: bool,
 
+    /// The parsed replay-event and replay-recording payloads.
+    /// NOTE: This is internal-only and not exposed into the Envelope.
+    #[serde(default, skip)]
+    replay_video_events: Option<(Vec<u8>, Vec<u8>)>,
```

Comment:
I would prefer if we don't put this on the header, but make it part of the item payload somehow. We should either

1. manage to encode all information into a single item payload (e.g. `{"some": "json"}\x00vide0_bytes`),
2. let them be separate items after all, and only combine them in `store.rs`.

@cmanallen let me know if you would prefer to discuss this in a meeting.

Reply:
Yes I would like to discuss in a meeting.